exercise3.js

```diff
@@ -0,0 +1,122 @@
+// Task 1. Add new Products: Google Pixel 6 Pro, Motorola Moto G31, Honor X7
+
+ products.push('Google Pixel 6 Pro', 'Motorola Moto G31', 'Honor X7')
+console.log(products)
+/*PS C:\Users\bisrat\Desktop\javascript1> node exercise3.js

```

Comment:
Task 1 is ok. Just delete these printouts.

Remove these printouts from all the tasks, to make the code more readable